### ISSUE TITLE
send_error_reports not accepted in the configuration and considered invalid YAML

### ISSUE THREAD

**Issue by uherting** [sabeechen/hassio-google-drive-backup#231]
I just added the following to the configuration and got a "invalid YAML" message:

`send_error_reports: true`

The documentation [README.md](https://github.com/sabeechen/hassio-google-drive-backup/blob/master/README.md) mentions this attribute with a default of `false`. 

Extract from the docs:
**send_error_reports** (default: `False`): When true, the text of unexpected errors will be sent to database maintained by the developer. This helps idenfity problems with new releases and provide better context messages when errors come up.

**Comment by uherting** [comment]
I cannot tell whether a duplicate was causing the problem as I did not make a screenshot.

After checking a snapshot prior to the day I issued this report I found that `send_error_reports: false` was contained in the addon.json. From this I assume that the entry I made was a duplicate key with a diffrent value though. 

Mea culpa! Excuse me for causing trouble.

FYI: Changing via the web-ui was possible and reflected in the configuration.
